backend/src/controllers/taskController.js

```diff
@@ -93,6 +93,51 @@ export default class TaskController {
     });
   }
 
+  //add subtask given taskID
+  saveSubtaskByTask(req) {
+    return new Promise((resolve, reject) => {
+      if (!req.body.subtask_title || !req.body.subtask_state || !req.params.taskId) {
+        return reject({ error: "Error with request body." });
+      }
+      const TaskModel = new Task();
+      const subtaskData = {
+        subtask_id: Math.floor(Math.random() * 100),
```

Comment:
let's pull subtask IDs from the body as well, instead of randomly generating a number. So coming from req.body.subtask_id